docs/content/user-guide.md

```diff
@@ -612,3 +612,7 @@ pk;import https://link/to/the/data/file.json
 
 ### Exporting your PluralKit data
 To export all the data associated with your system, run the `pk;export` command. This will send you a JSON file containing your system, member, and switch information.
+
+## Reply commands
+Commands that work with message links or ids will sometimes also work by replying to a message. If you use this method you do not need to use the message link/id field. You can tell if a command works with replies if it has <reply> as a field in the command in the [Command List](../commands). 
```

Comment:
Are there any commands that require a message argument that do not work with a reply?

Reply:
Currently no. And I guess if we made any that didn't we could change it.